src/defaults.mligo

```diff
@@ -4,15 +4,21 @@
 #include "types.mligo"
 #include "proposal.mligo"
 
```

Comment:
One more thing to check is that the `quorum_threshold` is between the `min_quorum_threshold` and `max_quorum_threshold` values.

Reply:
I've pushed an update to this now.